test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/ContextUrlWriterReaderTests.cs

```diff
@@ -871,11 +872,12 @@ public void ProjectedExpandedEntity()
         [Fact]
         public void ExpandedEntity()
         {
+
             ODataResource entry = new ODataResource()
             {
                 TypeName = TestNameSpace + ".Employee",
                 Properties = new[]
-                {

```

Comment:
>      [](start = 5, length = 5)

revert

Reply:
can you elaborate? What needs to be reverted?


---
In reply to: [256178335](https://github.com/OData/odata.net/pull/1402#discussion_r256178335) [](ancestors = 256178335)